src/cogent3/app/composable.py

```diff
@@ -946,7 +949,8 @@ def _call(self, val, *args, **kwargs):
     except Exception:
         result = NotCompleted("ERROR", self, traceback.format_exc(), source=val)
 
-    if not result and not isinstance(result, NotCompleted):
+    # an empty numpy array may be valid
```

Comment:
We have this comment, but there is no check for empty numpy array.

Reply:
The statement
```python
    # an empty numpy array may be valid
    if not is_np_array and not result and not isinstance(result, NotCompleted):
       ...
```
This is addressed by excluding numpy arrays from checking if `not result`